controller/TransparencyController.js

```diff
@@ -0,0 +1,50 @@
+const { Donation } = require("../services");
+const catchAsyncError = require("../middleware/catchAsyncError");
+
+/**
+ * @description get total donations
+ * @route GET /v1/transparency
+ * @access PUBLIC
+ */
+exports.getTotalAmount = catchAsyncError(async (req, res) => {
+  const startDate = new Date(req.query.startDate);
+  const endDate = new Date(req.query.endDate);
+
+  const totalDonations = await Donation.getTotal(startDate, endDate);
+
+  return res.status(200).json({
+    status: true,
+    data: totalDonations
+  });
+});
+
+/**
+ * @description get total donors and donations
+ * @route GET /v1/transparency
+ * @access PUBLIC
+ */
+exports.getTotalDonorsAndAmount = catchAsyncError(async (req, res) => {
+  const totalDonorsAndDonations = await Donation.getTotalDonorsAndAmount();
+
```

Comment:
Please wrap this in a `try... catch` block so we can properly manage the error state. See my comment above.